util/symbols.py

```diff
@@ -355,6 +416,18 @@ def retrieve_from_ranges(vram, rom=None):
         return None
 
 
+def mark_c_funcs_as_defined():
+    to_define = set(to_mark_as_defined)
```

Comment:
You should be able to use `to_mark_as_defined` directly instead of making a local set